lib/parallel_tests/tasks.rb

```diff
@@ -15,9 +15,7 @@ def purge_before_load
 
       def run_in_parallel(cmd, options={})
         count = " -n #{options[:count]}" unless options[:count].to_s.empty?
-        executable = File.expand_path("../../../bin/parallel_test", __FILE__)
-        command = "#{Shellwords.escape executable} --exec '#{cmd}'#{count}#{' --non-parallel' if options[:non_parallel]}"
-        command =  ParallelTests.with_ruby_binary(command)
+        command = "#{Shellwords.escape('parallel_test')} --exec '#{cmd}'#{count}#{' --non-parallel' if options[:non_parallel]}"
```

Comment:
don't need to shell-escape parallel_test ... shell-escapewas only there in case the path contained spaces

Reply:
I'm a little scared not using the full path could lead to weird scenarios where we try to run locally (like the tests) and it picks up the executable from the installed gem and not the local executable ... need to verify that does not happen ... or just keep the previous code ... which seems safer :>